chains/local_doc_qa.py

```diff
@@ -54,7 +56,9 @@ def seperate_list(ls: List[int]) -> List[List[int]]:
 def similarity_search_with_score_by_vector(
         self, embedding: List[float], k: int = 4
 ) -> List[Tuple[Document, float]]:
-    scores, indices = self.index.search(np.array([embedding], dtype=np.float32), k)
+    embeding_arr = np.array([embedding],dtype=np.float32)
+    embeding_arr = np.squeeze(embeding_arr)
```

Comment:
I'm thinking is it better to keep this squeeze logic to MyEmbedding because it seems like it is embedding implementation's responsibility to keep their dimension consistent.